eth/handler.go

```diff
@@ -820,6 +859,43 @@ func (h *handler) BroadcastBlock(block *types.Block, propagate bool) {
 	}
 }
 
+// needMoreDirectBroadcastPeers checks if the block should be broadcast to all direct peers
+// if the block is mined by self or received from proxyed validator, just broadcast to all direct peers
+// if not, just gossip it.
+func (h *handler) needMoreDirectBroadcastPeers(block *types.Block) bool {
+	if !h.enableBroadcastFeatures {
+		return false
+	}
+	parlia, ok := h.chain.Engine().(*parlia.Parlia)
+	if !ok {
+		return false
+	}
+	if parlia.ConsensusAddress() == block.Coinbase() {
```

Comment:
check mining, maybe It's a backup validator.

Reply:
It's ok, the backup validator won't produce block. Here only block producer need full broadcast to all other validators.